shared/src/vyxal/Context.scala

```diff
@@ -79,12 +79,12 @@ class Context private (
     * start of the list.
     */
   def pop(n: Int): Seq[VAny] =
-    if useStack then return parent.getOrElse(this).pop(n)
+    if useStack && parent.isDefined then return parent.get.pop(n)
```

Comment:
You might say that this is a change that should be tested, especially since we don't even know if an infinite loop is actually possible here (no one's reported such a bug). However, I would respond that I am tired and could not care less